tests/cxx/precomputed_tpl_args.cc

```diff
@@ -11,6 +11,8 @@
 
 // Tests the precomputed template-arg-use list in iwyu_cache.cc.
 
+#include <forward_list>
```

Comment:
`<forward_list>` seems a little confused about which commit it should be part of... C++17 special handling for it is introduced in the next commit, but the testcase is added here.

Reply:
It was not in the precomputed list at all, hence forward-declaration suggestions for contained type already worked for it. The second commit adds it to the list for pre-C++17 modes, and this test works in the default (C++17 currently) language standard.

Reply:
I see. It would look more symmetrical and balanced if you moved the `<forward_list>` testing together with the `<forward_list>` special-casing for pre-C++17, though, could you please transfer it to the [Add some more STL containers to precomputed list](https://github.com/include-what-you-use/include-what-you-use/pull/1326/commits/60a4ea0a3c3be1915519f17018551e3eaa7006cc) commit?

Reply:
Ok, sure.